releasenotes-dca/notes/ksm-pods-in-node-agent-13d9041602521d42.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The Kubernetes State Metrics (KSM) check can now be configured to collect
+    pods from the Kubelet in node agents instead of collecting them from the API
+    Server in the Cluster Agent or the Cluster check runners. This is useful in
+    clusters with a large number of pods where emitting pod metrics from a
+    single check instance might be too much.
```

Comment:
"might be too much" could be clarified. Too much for what?

Reply:
I changed this to add a bit more detail to try to clarify.